Pod/Classes/KolodaView/DraggableCardView/DraggableCardView.swift

```diff
@@ -233,42 +242,83 @@ public class DraggableCardView: UIView {
     }
     
     //MARK: Private
+    
+    private var directions: [SwipeResultDirection] {
+        return delegate?.card(cardAllowedDirections: self) ?? [.Left, .Right]
+    }
+    
     private var dragDirection: SwipeResultDirection {
-        return dragDistance.x > 0 ? .Right : .Left
+        //find closest direction
+        let normalizedDragPoint = dragDistance.normalizedDistanceForSize(bounds.size)
+        return directions.reduce((CGFloat.infinity, .None)) { min, dir in
```

Comment:
I think this shorthands works great in Geometry file, but here variables should be as descriptive as possible.